src/main/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCalls.java

```diff
@@ -353,11 +377,11 @@ private void generateIntervalsVCFFileFromAllShards() {
         final GermlineCNVIntervalVariantComposer germlineCNVIntervalVariantComposer =
                 new GermlineCNVIntervalVariantComposer(intervalsVCFWriter, sampleName,
                         refAutosomalIntegerCopyNumberState, allosomalContigSet);
-        germlineCNVIntervalVariantComposer.composeVariantContextHeader(getDefaultToolVCFHeaderLines());
+        germlineCNVIntervalVariantComposer.composeVariantContextHeader(sequenceDictionary, getDefaultToolVCFHeaderLines());
 
         logger.info(String.format("Writing intervals VCF file to %s...", outputIntervalsVCFFile.getAbsolutePath()));
         for (int shardIndex = 0; shardIndex < numShards; shardIndex++) {
-            logger.info(String.format("Analyzing shard %d / %d...", shardIndex, numShards));
+            logger.info(String.format("Analyzing shard %d / %d...", shardIndex + 1, numShards));
```

Comment:
Shards are named starting at 1 I take it?

Reply:
Thought I fixed this in another PR...dunno emoji